### DISCUSSION TITLE
Help - Understand how VS Code Marketplace search algorithms work?

### DISCUSSION THREAD

**Discussion by klink** [microsoft/vscode-discussions#870] [Extension Development QnA] [answered]
We have built a plugin for Vs Code https://marketplace.visualstudio.com/items?itemName=smallcloud.codify

We have several tags and categories listed in our plugin, when searching within them our plugin appear to be at the very bottom when sorting by "Relevancy", despite having much more downloads than the other plugins listed above. 

How can we optimise description and tags to appear higher in the "relevancy" section?

**Comment by alefragnani** [comment, marked as answer]
Hi @klink ,

I'm not sure how much of this information is available, but the VS Marketplace has its own repository (https://github.com/microsoft/vsmarketplace) for customer feedback and issue tracker, and I think you could find valuable information there. I remember a few threads about this topic in the past, and a quick look at the reported issues, I could find at least these two issues:

* https://github.com/microsoft/vsmarketplace/issues/71
* https://github.com/microsoft/vsmarketplace/issues/154

Maybe these, or other issues on that repo, may help you to improve to have a higher relevancy on search.

Hope this helps

**Reply by isidorn** [reply]
Aslo fyi @seaniyer
